scripts/install-systemd-multi-user.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
```

Comment:
nit: `/usr/bin/env bash` (same for others)

Reply:
fixed